### DISCUSSION TITLE
Overlay notification ?

### DISCUSSION THREAD

**Discussion by Fred-Vatin** [tauri-apps/tauri#7951] [Q&A]
I was wondering if it would be possible to display on overlay notification on desktop like the one sent by some keyboard keys state (capslock, num…).

![image](https://github.com/tauri-apps/tauri/assets/22337329/fe43334e-94e2-4402-a018-10cb2a37c3d0)

It should not take the focus or prevent user from writing. Any solution, at least on windows 10 ?

**Comment by Fred-Vatin** [comment]
Up ?

**Comment by FabianLars** [comment]
Did you try spawning a normal window with `decorations: false`, `ignoreCursorEvents: true` and `focus: false` ? I'm fairly sure that this is really close to what the notification on the screenshot does as well.
